airbyte-workers/src/test/java/io/airbyte/workers/internal/EmptyAirbyteSourceTest.java

```diff
@@ -302,8 +302,10 @@ public void testPerStreamWithMissingState() throws Exception {
     Assertions.assertThat(emptyAirbyteSource.isFinished()).isTrue();
   }
 
+  // In the LEGACY state, if the list of streams passed in to be reset does not include every stream
```

Comment:
let me know if these comments are confusingly written. just thought the tests should have some explanation.